trl/trainer/grpo_config.py

```diff
@@ -57,6 +55,21 @@ class GRPOConfig(TrainingArguments):
             capacity of a single GPU, albeit at the cost of slower generation. Disabling this option is not compatible
             with vLLM generation.
 
+        > Parameters that control generation
+
+        temperature (`float` or `None`, *optional*, defaults to `0.9`):
+            Temperature for sampling. The higher the temperature, the more random the completions.
+        top_p (`float` or `None`, *optional*, defaults to `None`):
+            Top-p value for sampling. If set to `None`, the default value will be used.
```

Comment:
let's expose the default values from `SamplingParams` instead

Reply:
For top_p that works, but I think `GenerationConfig` uses different defaults (also `top_k = -1` by default for vLLM, which I don't think is valid for HF generation)

Reply:
it should be good now